wiki/Game_mode/osu!catch/en.md

```diff
@@ -1,78 +1,84 @@
----
-needs_cleanup: true  # https://github.com/ppy/osu-wiki/issues/9918, also, needs a grammar/style check
----
+<!-- TODO: replace images to be taken from osu!(lazer) with the osu! argon skin.--> 
+
+<!-- Mostly since new players will touch lazer rather than stable in the future its just better to do it this way (this is also why the catcher is mostly referred to as 'platter', the argon skin uses a floating platter, not a character)-->
 
 # osu!catch
 
-![Gameplay of osu!catch](/wiki/shared/catch-gameplay.jpg "osu!catch Interface")
+osu!catch, formerly known as *Catch the Beat* or *CtB*, is a game mode that requires the player to control a platter that has the ability to *catch* falling fruits in tune to the beat.
 
-osu!catch, formerly known as *Catch the Beat* or *CtB*, is a game mode that requires the player to control a character(catcher) who catches falling fruit in tune to the beat.
+## Song Selection
 
-## ![](/wiki/shared/mode/catch.png) Gameplay
+To access the osu!catch game mode press `Ctrl` + `3` at the same time, or use the the `Mode` button and select `osu!catch` in osu!(stable) or click the ![](/wiki/shared/mode/catch.png) icon on the top of the screen for osu!(lazer).
 
-### Song Selection
+To download more beatmaps for osu!catch, head to the [beatmap listing](https://osu.ppy.sh/beatmapsets) on the osu! website, or while in osu!(lazer) press `Ctrl` + `B` or click the `Beatmap listing` button 
 
-To access the osu!catch game mode, press `Ctrl` + `3` at the same time.
+![osu!(lazer) beatmap listing button](/wiki/shared/lazer_beatmap_listing.png)
 
-Alternatively, click on the `Mode` button and select `osu!catch`.
+## Player control
 
-### Gameplay Basics
+In osu!catch, the player is supposed to catch falling fruits by controlling a **platter** (also known as a "catcher") to the rhythm of the music. By default the keys to control the platters movements are:
 
-#### Playfield
+- the left `<-` and right `->` or `Z` and `X` keys to move the catcher left and right respectively.
+- Press and hold the `leftShift` key or the Left Mouse Button to activate a *dash* state which is recognizable by it making the platter *glow brightly* and leaving an afterimage while moving. Moving while in the dash state makes the player move **2 times faster** then normally.
```

Comment:
`Shift` is usually written as so and i don't think any other article distinguishes between the left and right keys

```suggestion
- Press and hold the left `Shift` key or the left mouse button to activate a *dash* state which is recognisable by it making the platter *glow brightly* and leaving an afterimage while moving. Moving while in the dash state is **twice as fast** than normal.
```